### PR TITLE
Improved: UI of Settings page(#32j3r6t)

### LINKED ISSUE #93: Update Settings page with updated UI
Reference - https://www.figma.com/file/bVPRRw282CqGKMdbz7dciH/HC-Ionic-design-system?node-id=23579%3A83419&t=EwnGrKo71tXMohaa-0

### PR DESCRIPTION
**Pull request by shashwatbangar**
### Related Issues

Closes #93 

### Screenshots of Visual Changes before/after (If There Are Any)

Screenshot 2023-01-17 at 3 00 19 PM

**IMPORTANT NOTICE** - Remember to update `CHANGELOG.md` with description of your change

### Contribution and Currently Important Rules Acceptance

- [ ] I read and followed [contribution rules](https://github.com/DivanteLtd/vsf-capybara/blob/master/CONTRIBUTING.md)

### CODE REVIEW

**Review by k2maan — APPROVED** [review]

**Review by disha1202 — APPROVED** [review]

File: src/views/Settings.vue [hunk 1]
```diff
@@ -6,38 +6,83 @@
...
+          </ion-card-content>
+          <ion-button @click="goToOms" fill="clear">
+            {{ $t('Go to OMS') }}
+            <ion-icon slot="end" :icon="openOutline" />
+          </ion-button>
+        </ion-card>
+        <ion-card>
+          <ion-card-header>
+            <ion-card-subtitle>
+              {{ $t("Shopify Config") }}
```
Comment (adityasharma7) [inline src/views/Settings.vue:47]: We do not have ecommerce or shopify config in this app, we have facilities that the user is associated with. Title and content should be accordingly

**Review by adityasharma7 — CHANGES_REQUESTED** [review]

File: src/views/Settings.vue (hunk 1 above)
Comment (shashwatbangar) [reply src/views/Settings.vue:47]: Sure sir. Updating it accordingly.

File: src/locales/en.json [hunk 2]
```diff
@@ -5,8 +5,12 @@
   "Complete": "Complete",
   "Complete picklist": "Complete picklist",
   "Completed": "Completed",
+  "eCommerce": "eCommerce",
+  "eCommerce stores are directly connected to one Shop Configs. If your OMS is connected to multiple eCommerce stores selling the same catalog operating as one Company, you may have multiple Shop Configs for the selected Product Store.": "eCommerce stores are directly connected to one Shop Configs. If your OMS is connected to multiple eCommerce stores selling the same catalog operating as one Company, you may have multiple Shop Configs for the selected Product Store.",
```
Comment (adityasharma7) [inline src/locales/en.json:9]:
```suggestion
```

File: src/locales/en.json [hunk 3]
```diff
@@ -24,12 +29,17 @@
   "Product not found": "Product not found",
   "Scan": "Scan",
   "Select all" : "Select all",
+  "Select eCommerce": "Select eCommerce",
```
Comment (adityasharma7) [inline src/locales/en.json:32]:
```suggestion
```

**Review by adityasharma7 — CHANGES_REQUESTED** [review]

File: src/locales/en.json [hunk 4]
```diff
@@ -24,12 +27,16 @@
   "Product not found": "Product not found",
   "Scan": "Scan",
   "Select all" : "Select all",
+  "Select facility": "Select facility",
   "Settings": "Settings",
+  "Shopify Config": "Shopify Config",
```
Comment (adityasharma7) [inline src/locales/en.json:32]:
```suggestion
```

**Review by adityasharma7 — APPROVED** [review]

**Pull request merged**